package.json

```diff
@@ -40,7 +40,7 @@
         "sprintf-js": "^1.1.2",
         "static-kdtree": "^1.0.2",
         "sweetalert": "^2.1.2",
-        "three": "^0.90.0",
+        "three": "^0.111.0",
```

Comment:
Have you tried [running tests](https://github.com/Exabyte-io/wave.js#tests) with the new Three.js version and it was successful?

Reply:
Running test of master branch went well with  the new Three.js version.
But, that  of add_axis_labels branch is  not successful.

I am looking for the cause of the following error:
  ● Test suite failed to run

    /home/msaito/GitHub/wave.js/node_modules/three/examples/jsm/renderers/CSS2DRenderer.js:5
    import {
    ^^^^^^
    SyntaxError: Cannot use import statement outside a module

Reply:
JSET seems to be unable to import  ES6 modules.
Although I  tried to modified .babelrc to be able to import ES6 modules, it didn't improve.


If the "three-css2drender" package is used as in the beginning https://github.com/Exabyte-io/wave.js/pull/38, the test is successful.